action.yml

```diff
@@ -151,6 +161,21 @@ runs:
         fi
 
         pip install "${repo_url}/@${{inputs.nuitka-version }}#egg=nuitka"
+    - name: Install ccache
+      if: ${{ inputs.disable-cache != 'ccache' && runner.os == 'Linux' }}
+      shell: bash
+      run: |
+        sudo apt-get install -y ccache
+    - name: Cache Nuitka cache directory
+      if: ${{ !inputs.disable-cache }}
```

Comment:
Very well, partial disable won't work that way, but I guess, that's almost perfect.